src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.html

```diff
@@ -1,14 +1,69 @@
+<xuilib-hmcts-error-summary [errorMessages]="validationErrors" [header]="'There is a problem'"></xuilib-hmcts-error-summary>
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-three-quarters">
-    <exui-hearing-parties-title></exui-hearing-parties-title>
-    <h1 class="govuk-heading-l">
-      Do you require any additional facilities?
-    </h1>
-    <exui-case-flags
-      [info]="'Any active flags on this case may require additional facilities at the hearing.'"
-      [hearingValueModel]="serviceHearingValuesModel"
-      [caseFlagsRefData]="caseFlagsRefData"
-      [caseFlagType]="caseFlagType">
-    </exui-case-flags>
-  </div>
+  <form class="form" [formGroup]="hearingFactilitiesForm">
+    <div class="govuk-grid-column-three-quarters">
+      <exui-hearing-parties-title></exui-hearing-parties-title>
+      <h1 class="govuk-heading-l">
+        Do you require any additional facilities?
+      </h1> 
+      <exui-case-flags
+        [info]="'Any active flags on this case may require additional facilities at the hearing.'"
+        [hearingValueModel]="serviceHearingValuesModel"
+        [caseFlagsRefData]="caseFlagsRefData"
+        [caseFlagType]="caseFlagType">
+      </exui-case-flags>
+    </div>
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-form-group govuk-!-margin-top-4" [ngClass]="{'govuk-form-group--error': !additionSecurityRequiredValid}">
+          <fieldset class="govuk-fieldset"  id="addition-security-confirmation">
+              <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+                  <h3 class="govuk-fieldset__heading">
+                      Will additional security be required?      &nbsp;  &nbsp; &nbsp; &nbsp;
```

Comment:
are these random &nbsp; needed?

Reply:
Could please just say you what you want me to do? How can they be random. They were taken from design. @rayliangatos  what is your stance on using @nbspc I have seen them in @Kartheek-S work and nothing was said. We have closing the sprint. We do not have time for this @rayliangatos.

Reply:
nbsp is only needed when it's needed, but this case it doesn't, though it's in the prototype we should remove it.